quickref/qsharp-quick-reference.tex

```diff
@@ -342,6 +609,73 @@
     Run all unit tests                & \texttt{dotnet test} \\
   \end{keysref}
 
+	\columnbreak
+	\section{Math reference}
+
+	\begin{keysref}{Complex Arithmetic}
+    	\(i^2 \) & \(-1\) \\
+	\( (a + bi) + (c + di) \) & \( (a + c ) + (b + d)i \) \\
+	\( (a + bi)  (c + di) \) & \( a \cdot b + a \cdot di + c \cdot bi + (b \cdot d)i^2 \) \newline
+		\(  a \cdot b + a \cdot di + c \cdot bi - (b \cdot d) \) \\
+
+	Complex Conjugate \newline \(x = a + bi \) & \( \overline{x} = a - bi \) \\
+ 	Complex division \newline \( \frac{a +bi}{c+di} \) & \( \frac{x}{y} =  \frac{x}{y} \cdot 1 = \frac{x}{y} \cdot \frac{\overline{y}}{\overline{y}} \) \newline
+		\( = \frac{(a+bi)(c-di) }{c+di)(c-di)} = \frac{(a+bi)(c+di)}{c^2 + d^2} \) \\
+	Modulus \newline \( x = a + bi \) & \( |x| = \sqrt{a^2 + b^2} \) \\
+	\(e^{i\theta} \) & \( \cos(\theta)  + i \sin(\theta) \) \\ 
+ 	\( e^{a + bi} \) & \( e^a + e^{bi} = e^a(\cos(b) + i\sin(b)) \)  \\
+	\( r^{a+bi} \) & \( c^a \cdot c^{bi} = c^a \cdot e^{bi\ln(c)} \) \newline \(= c^a((\cos(b\ln(c)) + i\sin(b\ln(c))) \)\\
+	Polar form \newline \(a+bi \) & \( re^{i\theta} = r(\cos(\theta) + i\sin(\theta))  \) \newline
+		\( r =  \sqrt{a^2 + b^2} \) \newline \( \theta = \tan^{-1}(\frac{b}{a}) \) \\
+	\end{keysref}
+
+	\begin{keysref}{Linear Algebra}
+	\usebox\matrixadditionleft & \usebox\matrixadditionright \\
+	
+	\usebox\matrixscalarleft &\usebox\matrixscalarright \\
+	
+	\usebox\matrixmultiplicationleft & \usebox\matrixmultiplicationright \\
+
+	\usebox\matrixtransposeleft & \usebox\matrixtransposeright \\	 
+	
+	\usebox\matrixadjointleft & \usebox\matrixadjointright \\
+
+	\usebox\innerproductleft & \usebox\innerproductright \\ 	
+	
+	\usebox\outerproductleft & \usebox\outerproductright \\
+	
+\end{keysref}
+
+\newpage
+
+\section{Gates reference}
+
+
+\begin{keysrefQbits}{Single Qubit gates}
+
+	Gate & Matrix representation    & Ket-Bra & Applying to \( |\psi\rangle = \alpha|0\rangle + \beta|1\rangle \) & Applying to basis states & & \\
```

Comment:
I would list basis states here, 0 and 1 are very common notations, + and - slightly less, and i and -i even less common:
```suggestion
	Gate & Matrix representation    & Ket-bra representation & Applying to \( \ket{\psi} = \alpha\ket{0} + \beta\ket{1} \) & Applying to basis states: & $\ket{0},\: \ket{1},\: \ket{+},\: \ket{-}\:$ and  & $\ket{\pm i} = \frac{1}{\sqrt2}(\ket{0} \pm i\ket{1})$ \\
```